### PR TITLE
Bugfixes/user account related

### PR DESCRIPTION
**Pull request by wlteo**
Added a bunch of fixes. Please see commit history for full story.

### CODE REVIEW

File: app/views/tweets/main.html.erb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <div class="container">
   <div class="row">
-  </div>
-  <div class="row">
+    <div class="col-md-6 col-offset-6">
```
Comment (gohdaniel15) [inline app/views/tweets/main.html.erb]: Col-md-offset-6

File: app/controllers/tweets_controller.rb [hunk 2]
```diff
@@ -1,12 +1,14 @@
 class TweetsController < ApplicationController
   def main
```
Comment (gohdaniel15) [inline app/controllers/tweets_controller.rb]: Prefer a more descriptive name. Maybe home?

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,15 @@
...
+  username = Faker::Pokemon.name
+  email = Faker::Internet.email
+  User.create(username: username, email: email, password: "password123", is_admin: 0)
+end
+
+10.times do
+  body = Faker::ChuckNorris.fact
+  user_id = Faker::Number.between(1, 10)
+  Tweet.create(body: body, user_id: user_id)
+end
```
Comment (gohdaniel15) [inline db/seeds.rb:19]: Nice

**Review by gohdaniel15 — APPROVED** [review]
Good work!

**Review by sabrina-s — APPROVED** [review]
Good to go after Daniel's suggested changes! 🚀

**Review by sabrina-s — APPROVED** [review]
Good to go! 🚀

**Pull request merged**